dash/development/component_loader.py

```diff
@@ -35,6 +36,14 @@ def load_components(metadata_path, namespace="default_namespace"):
     `type`, `valid_kwargs`, and `setup`.
     """
 
+    warnings.warn(
+        DeprecationWarning(
+            "Dynamic components loading has been deprecated and will be removed"
+            " in dash 3.0.\n"
```

Comment:
is there an issue filed to removed dynamic component loading so we don't forget?